torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -505,13 +506,7 @@ def forward(self, *inputs: Any) -> Any:
                 aug = self._apply_image_or_video_transform(
                     aug, transform_id, magnitude, interpolation=self.interpolation, fill=self.fill
                 )
-            mix.add_(
-                # The multiplication below could become in-place provided `aug is not batch and aug.is_floating_point()`
-                # Currently we can't do this because `aug` has to be `unint8` to support ops like `equalize`.
-                # TODO: change this once all ops in `F` support floats. https://github.com/pytorch/vision/issues/6840
-                combined_weights[:, i].reshape(batch_dims)
-                * aug
-            )
+            mix.add_(combined_weights[:, i].reshape(batch_dims) * aug)
```

Comment:
Is the condition of the previous comment "aug.is_floating_point" met here?

Reply:
I don't understand the question. Aug can be float or int but it's not modified in place, so we don't care about its type. Also note there is no code change here, just removing the comment and linting.

Reply:
Oh sorry, I misunderstood. So you are just removing the comment and keeping the code as is, right? So this is just doing what you proposed in https://github.com/pytorch/vision/pull/6861/files#r1008326597, right?

Reply:
Correct! All good! Just wanted to confirm you didn't mean something different. :)